sdk/identity/identity/test/manual-integration/AzureVM/INSTRUCTIONS.md

```diff
@@ -0,0 +1,126 @@
+# prerequisite tools
+- Azure CLI
```

Comment:
This is great as an internal thing, but seems like some of this could be useful for users! Do we have a plan on the long run to make this more user friendly?

Reply:
Yes, we can continue to polish this as more people use it and turn it into docs. This PR is one step in that direction

Reply:
That sounds great 🌞